spec/unit/block_score/not_found_error_spec.rb

```diff
@@ -1,10 +1,9 @@
-module BlockScore
-  RSpec.describe NotFoundError do
-    subject { -> { BlockScore::Person.retrieve('abc123') } }
-    let(:message) do
-      '(Type: invalid_request_error) Person with id abc123 could not be found (Status: 404)'
-    end
-
-    it_behaves_like 'an error'
+RSpec.describe BlockScore::NotFoundError do
+  subject { -> { BlockScore::Person.retrieve('abc123') } }
```

Comment:
Instance of https://github.com/BlockScore/blockscore-ruby/pull/68/files#r52838287